### PR TITLE
Add make deploy

### PR DESCRIPTION
**Pull request by esmet**
Make local development easier by adding a make deploy target.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,6 +20,7 @@ unexport SSH_CLIENT
 unexport SSH2_CLIENT
 
 NAME ?= ambassador
+DEV_KUBECONFIG ?= ~/.kube/config
```
Comment (acookin) [inline Makefile:23]: same feedback, this should probaly just fail and tell the user to export a DEV_KUBECONFIG

**Review by acookin — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (kflynn) [reply Makefile:23]: I see the `preflight-dev-kubeconfig` target, which is great! need to drop this assignment too. 😉

**Review by kflynn — CHANGES_REQUESTED** [review]
One last tweak. 🙂 Thanks!

File: Makefile (hunk 1 above)
Comment (esmet) [reply Makefile]: Done

**Review by acookin — APPROVED** [review]

**Review by kflynn — APPROVED** [review]
Hit it! Thanks. 🙂

**Pull request merged**